lib/liquid/combinators/tags/if.ex

```diff
@@ -68,6 +68,6 @@ defmodule Liquid.Combinators.Tags.If do
       parsec(:variable_definition)
     ])
     |> optional(times(parsec(:logical_condition), min: 1))
-    |> tag(:if_condition)
+    |> tag(:evaluation)
```

Comment:
`evaluation` is too much generic. An evaluation can be anything. A better name can be `boolean_condition` or simply `condition` https://en.wikipedia.org/wiki/Conditional_(computer_programming)